AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -43,16 +40,57 @@ export var functionsFlow = {
 
         if (!await isKuduAccessiblePromise)
         {
-            if (siteInfo.kind.includes("linux") || siteInfo.kind.includes("container")){
-                flowMgr.addView(new CommonWordings().connectivityCheckUnsupported.get());
-            }else{
-                flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
-            }
+            flowMgr.addView(new VnetDnsWordings().cannotCheckWithoutKudu.get("Functions settings"));
             return;
         }
-
-        var isDaasExtAccessible = await diagProvider.checkDaasExtReachable(30);
-
+        var param = [];
+        param.push("api-version=2015-08-01");
+        var daasVersionInfo=await diagProvider.getDaaSExtApiAsync("daasversion",param); 
+        if(daasVersionInfo.status == "200")
+        {
+            var isDaasExtAccessible= true;
+        }
+        else
+        {
+            var isDaasExtAccessible= false;
+        }  
+             
+        var daasVersion=daasVersionInfo.body.Version;        
+        var isDaasNew=false; 
+        if(isDaasExtAccessible == true){
+            function versionCompare(v1, v2) 
+            { 
+                var vnum1 = 0, vnum2 = 0; 
+                for (var i = 0, j = 0; (i < v1.length 
+                                        || j < v2.length);) { 

```

Comment:
move "{" to next line for consistent formatting

Reply:
fixed